### PR TITLE
Hotfix/resolving reported issues

### PR DESCRIPTION
**Pull request by CiotkaCierpienia**

### CODE REVIEW

**Comment by trzcina** [comment]
Hi, I've checked updates - it is clear and very helpful.
One thing for the readability - maybe we should add this header:
How-to-update-Content-Type-Definitions-Flotiq-docs

**Comment by andrzejwp** [comment]
LGTM

File: docs/API/content-type/creating-co.md [hunk 1]
```diff
@@ -3,106 +3,392 @@ description: How to add Content Objects in Flotiq
...
-Once a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> has been defined in the system - the user can create <abbr title="Content Object - an instance of a Content Type.">Content Objects</abbr>  of that Content Type. This is done either directly through the API or via the convenient Content Entry tools provided within the Content Management Platform.
+Once a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> 
+has been defined in the system - the user can create 
+<abbr title="Content Object - an instance of a Content Type.">Content Objects</abbr>  of that Content Type. 
+This is done either directly through the API or via the convenient Content Entry tools provided within the 
+Content Management Platform.
 
-## Authoring content through the API
+!!! note
+    You will need to use your `Application Read and write API KEY` to perform this action
```
Comment (trzcina) [inline docs/API/content-type/creating-co.md:13]: I don't know what does it mean by "this action" in this context.

File: docs/API/content-type/creating-co.md [hunk 2]
```diff
@@ -3,106 +3,392 @@ description: How to add Content Objects in Flotiq
...
+        }
+        ```
+
+#### Possible validation errors
+
+| Error                                        | Description                                                                                               |
+| -------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| This value is already used                   | Send when the property has to be unique, and the value is already used in an existing object of that type |
+| Must be at least 1 characters long           | Send when the property is required, and an empty string was sent in object                                |
+| The property {name} is required              | Send when the property is required and was missing in the sent object                                     |
```
Comment (trzcina) [inline docs/API/content-type/creating-co.md:248]: Unfortunatelly this is rendered as 
image

Comment (trzcina) [reply docs/API/content-type/creating-co.md:248]:
```suggestion
| The property \{name\} is required              | Send when the property is required and was missing in the sent object                                     |
```

File: docs/API/content-type/creating-co.md (hunk 1 above)
Comment (trzcina) [reply docs/API/content-type/creating-co.md:13]: Ok, it's about creating content objects, I think the title of the page "Content Objects" could be "Creating new Content Objects".

File: docs/API/content-type/creating-ctd.md [hunk 3]
```diff
@@ -1,19 +1,307 @@
...
+In this case, Blog Post will have `title` property which will be unique and will render as text input in the CMS panel 
+and `postContent`, which can be duplicated and presented as CKEditor input in the CMS panel. 
+The first input in the form in the CMS panel will be `title` input, and the second will be `postContent`.
+
+!!! Example
+
+    === "CURL"
+    
+        ```
+        curl -X POST "https://api.flotiq.com/api/v1/internal/contenttype" -H 'accept: */*' -H 'X-AUTH-TOKEN: YOUR_API_KEY' -H 'Content-Type: application/json' --data-binary '{"name":"blogposts","label":"Blog Posts","schemaDefinition":{"type":"object","allOf":[{"$ref":"#/components/schemas/AbstractContentTypeSchemaDefinition"},{"type":"object","properties":{"title":{"type":"string"},"postContent":{"type":"string"}}}],"required":["title","postContent"],"additionalProperties":false},"metaDefinition":{"propertiesConfig":{"title":{"label":"Title",inputType":"text","unique":true},"postContent":{"label":"Post content","inputType":"richtext","unique":false}},"order":["title","postContent"]}}'
```
Comment (trzcina) [inline docs/API/content-type/creating-ctd.md:122]: When i use this example code i get: `{"data":["Invalid json message received"]}`
There is a typo here
Hotfix-resolving-reported-issues-by-CiotkaCierpienia-·-Pull-Request-60-·-flotiq-flotiq-docs

File: docs/API/content-type/creating-co.md [hunk 4]
```diff
@@ -3,106 +3,392 @@ description: How to add Content Objects in Flotiq
...
+(where `name` is the name of the content type definition) to create a new Content Object:
 
-
-For a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> defined according to the example described above, a very simple POST payload can be sent to the supporting endpoint to create a new Content Object:
-
-``` json
+```
 {
-  "id": "123123123",
   "title": "New object",
```
Comment (trzcina) [inline docs/API/content-type/creating-co.md:26]: Removing id affects next examples on batch (updateExisting) which presents update based on `id`; I will add "Let's assume that we have blogpost with id `123123123123` to the batch section.

File: docs/API/content-type/creating-ctd.md [hunk 5]
```diff
@@ -60,119 +350,610 @@ Creating a new <abbr title="Content Type - a model of data that has been defined
...
+| schemaDefinition.allOf.1.$ref                    | `Invalid schema definition for content type. Schema definition should have object with properties list. Check https://flotiq.com/docs/API/content-types/creating-ctd/ for details.`                                                                                       | Send when `schemaDefinition` property does not have properties definitions in the first object of `allOf` array                                                        |
+| schemaDefinition.allOf[1].properties.{name}.type | `Does not have a value in the enumeration [\"array\", \"boolean\", \"integer\", \"null\", \"number\", \"object\", \"string\"]`                                                                                                                                            | Send when property type is invalid, for valid properties type check [properties table](#property-types)                                                                |
```
Comment (trzcina) [inline docs/API/content-type/creating-ctd.md:406]: Invalid rendering properties with curly braces

File: docs/API/content-type/creating-ctd.md [hunk 6]
```diff
@@ -60,119 +350,610 @@ Creating a new <abbr title="Content Type - a model of data that has been defined
...
+| ------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name                                             | `This value is already used`                                                                                                                                                                                                                                              | Send when Content Type with that type already exists                                                                                                                   |
```
Comment (trzcina) [inline docs/API/content-type/creating-ctd.md]:
```suggestion
| name                                             | `This value is already used`                                                                                                                                                                                                                                              | Send when Content Type with that name already exists                                                                                                                   |
```

File: docs/API/content-type/creating-ctd.md [hunk 7]
```diff
@@ -60,119 +355,610 @@ Creating a new <abbr title="Content Type - a model of data that has been defined
...
+| -------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name                                               | `This value is already used`                                                                                                                                                                                                                                              | Send when Content Type with that type already exists                                                                                                                   |
```
Comment (trzcina) [inline docs/API/content-type/creating-ctd.md:399]:
```suggestion
| name                                               | `This value is already used`                                                                                                                                                                                                                                              | Send when Content Type with that name already exists                                                                                                                   |
```

File: docs/API/content-type/deleting-co.md [hunk 8]
```diff
@@ -4,27 +4,195 @@ description: How to delete Content Objects in Flotiq
...
-* `object ID` is the ID of the object to remove
-
-For example: 
-
-```
-curl -X DELETE "https://api.flotiq.com/api/v1/content/blogposts/blogposts-1" -H "accept: application/json" -H "X-AUTH-TOKEN: YOUR_API_TOKEN"
-``` 
-
-For successful delete Flotiq will return `204 OK` response, for non-existing objects `404 Not Found` response, and for objects in relation within another object `400 Validation error` response.
+Deleting of the object is done as the soft delete. All items will be still in the database, but not accessible by the API.
```
Comment (trzcina) [inline docs/API/content-type/deleting-co.md:7]: To be discussed how to tell the user that his data can be restored from backup etc.
This section should be moved to the bottom of this page (it's an edge case).

File: docs/API/content-type/listing-co.md [hunk 9]
```diff
@@ -1,61 +1,991 @@
...
+defined according to the [create Content Type example](/API/content-type/creating-ctd), a very simple `GET` request can be sent
+to the supporting endpoint `https://api.flotiq.com/api/v1/content/{name}`
+(where `name` is the name of the content type definition) to get Content Objects.
+
+!!! Example
+
+    === "CURL"
+
+        ``` 
+        curl -X GET "https://api.flotiq.com/api/v1/content/blogposts?page=1&limit=20&order_by=internal.createdAt&order_direction=asc&hydrate=0&filters=%7B%7D" -H "X-AUTH-TOKEN: YOUR_API_KEY" -H "accept: application/json"
```
Comment (trzcina) [inline docs/API/content-type/listing-co.md:29]: When I copy and run this example I get the error "malformed filters"

File: docs/API/content-type/listing-co.md [hunk 10]
```diff
@@ -1,61 +1,991 @@
...
     }
-  ]
-}
-```
+    ```
+
+    === "CURL"
+    
+        ```
+        curl -X GET "https://api.flotiq.com/api/v1/content/blogposts?page=1&limit=20&order_by=internal.createdAt&order_direction=asc&hydrate=0&filters=%7Btitle%3A%7Btype%3A%22equals%22%2Cfilter%3A%22New%20object%22%7D%2CpostContent%3A%7Btype%3A%22contains%22%2Cfilter%3A%22content%22%7D%7D" -H "X-AUTH-TOKEN: YOUR_API_KEY" -H "accept: application/json"
```
Comment (trzcina) [inline docs/API/content-type/listing-co.md:198]: When I copy and run this example I get the error "malformed filters"

Comment (trzcina) [reply docs/API/content-type/listing-co.md:198]: This string decoded contains `"{title:{type:"equals",filter:"New object"},postContent:{type:"contains",filter:"content"}}"`
It should be JSON, with object keys quoted

File: docs/API/content-type/listing-co.md [hunk 11]
```diff
@@ -1,61 +1,991 @@
...
+            .asString();
+        ```
+
+    === "Node + Request"
+      
+        ```
+        const request = require('request');
+
+        const options = {
+            method: 'DELETE',
```
Comment (trzcina) [inline docs/API/content-type/listing-co.md:268]: It should be GET method, not DELETE

File: docs/API/content-type/listing-co.md [hunk 12]
```diff
@@ -1,61 +1,991 @@
...
+            url: 'https://api.flotiq.com/api/v1/content/blogposts',
+            qs: {
+                page: '1',
+                limit: '20',
+                order_by: 'internal.createdAt',
+                order_direction: 'asc',
+                hydrate: '0',
+                filters: '{title:{type:"equals",filter:"New object"},postContent:{type:filter:"content"}}'
```
Comment (trzcina) [inline docs/API/content-type/listing-co.md]: Invalid json

File: docs/API/content-types.md [hunk 13]
```diff
@@ -52,21 +52,33 @@ You can learn how to obtain your API key [here](index.md).
...
 
-While working with Content Objects, you can:
+The supporting endpoints of a given
+<abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr>
+allow the user to perform basic REST operations
+
+![](images/endpoints.png){: .center .width75 .border}
 
 * [Create](content-type/creating-co.md)
     * [Batch create](/API/content-type/creating-co/#batch-content-upload)
```
Comment (trzcina) [inline docs/API/content-types.md:75]:
```suggestion
    * [Batch create](/API/content-type/creating-co/#batch-create-content-objects-through-api)
```

File: docs/API/content-types.md [hunk 14]
```diff
@@ -52,21 +52,33 @@ You can learn how to obtain your API key [here](index.md).
...
 * [Update](content-type/updating-co.md)
     * [Batch update](/API/content-type/creating-co/#batch-content-upload)
```
Comment (trzcina) [inline docs/API/content-types.md:77]:
```suggestion
    * [Batch update](/API/content-type/creating-co/#batch-create-content-objects-through-api)
```

File: docs/API/content-type/deleting-co.md (hunk 8 above)
Comment (trzcina) [reply docs/API/content-type/deleting-co.md]: This section will be moved to separate page

**Pull request merged**